Rename file extension of RunnerTemplate.Windows.txt to .cmd

akoeplinger opened the pull request:
So it automatically gets the EOL treatment from #1336.

---

Original description:

After https://github.com/dotnet/runtime/pull/1253 was merged I got the following error when running build-native.cmd on my machine:

```
C:\Users\Alexander\Dev\runtime> src\libraries\Native\build-native.cmd x64 Debug Windows_NT outconfig netcoreapp5.0-Windows_NT-Debug-x64
**********************************************************************
** Visual Studio 2019 Developer Command Prompt v16.5.0-pre.2.0
** Copyright (c) 2019 Microsoft Corporation
**********************************************************************
[vcvarsall.bat] Environment initialized for: 'x86_x64'
Commencing build of native components

The system cannot find the batch label specified - GenVSSolution
```

Adding a comment before the exit call fixes this though I'm not fully sure why. Maybe the CMD parser got confused.

akoeplinger commented:
Hmm I'm seeing this in more places, e.g. RunTests.cmd too. Sounds like my machine might have an issue or has anyone else seen this?

jkotas commented:
I have not seen it, but I know that .cmd files tend to break in unpredictable ways like this one.

@danmosemsft Sounds like there is something wrong with #1253 ...

tannergooding commented:
This may be the same reason as why I did https://github.com/dotnet/runtime/pull/1336 and modifying the file just added the needed crlf further label to be correctly found.

Recloning the repo to see if that resolves the issue may be a good idea.

akoeplinger committed:
Rename file extension of RunnerTemplate.Windows.txt to .cmd

So it automatically gets the EOL treatment from https://github.com/dotnet/runtime/pull/1336.

akoeplinger commented:
@tannergooding good catch, that was indeed the issue! I found that the RunnerTemplate.Windows.txt was still LF though since it doesn't have .cmd extension, I fixed that in this PR.

jkotas reviewed (approved):
Thanks!

jkotas commented on eng/testing/tests.targets:
```diff
@@ -3,8 +3,8 @@
 
-    <RunScriptInputName Condition="'$(TargetOS)' == 'Windows_NT'">RunnerTemplate.Windows.txt</RunScriptInputName>
-    <RunScriptInputName Condition="'$(TargetOS)' != 'Windows_NT'">RunnerTemplate.Unix.txt</RunScriptInputName>
+    <RunScriptInputName Condition="'$(TargetOS)' == 'Windows_NT'">RunnerTemplate.Windows.cmd</RunScriptInputName>
```
Nit: We do not need the "Windows" and "Unix" in the name anymore

akoeplinger committed:
Review feedback
